nmma/em/analysis.py

```diff
@@ -386,21 +391,6 @@ def get_parser(**kwargs):
         help="Number of cos-iota nodes used in the Bayestar fits (default: 10)",
         default=10,
     )
-    parser.add_argument(
```

Comment:
your PR does not contains these flags (ra, dec, dL). is it something that you want?